_includes/index-doc-item.html

```diff
@@ -1,17 +1,21 @@
 {% assign is_external_guide = include.url | startswith: 'http' %}
 {% if is_external_guide %}
-  {% assign guide_url = include.url %}
+{% assign guide_url = include.url %}
 {% elsif include.docversion == 'latest' %}
-  {% assign guide_url = site.baseurl | append: include.url %}
+{% assign guide_url = site.baseurl | append: include.url %}
 {% else %}
-  {% assign guide_url = site.baseurl | append: '/version/' | append: include.docversion | append: include.url %}
+{% assign guide_url = site.baseurl | append: '/version/' | append: include.docversion | append: include.url %}
 {% endif %}
-<div class="{% if include.origin == 'quarkiverse-hub' %}quarkiverse {% endif %}grid__item docs-card {{ include.class }}">
-  <h4><a href="{{ guide_url }}" {% if is_external_guide %}target="_blank" {% endif %}>{{ include.title }}</a></h4>
-  <div class="description">{{ include.summary | markdownify }}</div>
-  <div class="keywords">{{ include.keywords }}</div>
-  <div class="categories">{{ include.categories }}</div>
-  {% if include.origin == 'quarkiverse-hub' -%}
-  <div class="origin">Quarkiverse Hub</div>
-  {% endif -%}
-</div>
+<qs-guide type="{{ include.type | escape }}"
+          title="{{ include.title | escape }}"
+          url="{{ guide_url }}"
+          summary="{{ include.summary | markdownify }}"
+          keywords="{{ include.keywords | escape }}"
+          categories="{{ include.categories | escape }}"
+          origin="{{ include.origin | escape }}"
+>
+    <!-- WE KEEP PART OF THE CONTENT FOR SEO -->
```

Comment:
Why not keep the whole content while we're at it? It would be easier for debugging quarkus.io when one doesn't have an instance of search.quarkus.io running locally.

Reply:
I am not sure I understand the plus of adding the whole content. It would make the page longer to load, also it's not visible anyway.

Reply:
> It would be easier for debugging quarkus.io when one doesn't have an instance of search.quarkus.io running locally.

But okay, fair point about the page being longer to load.